standalone.tpl

```diff
@@ -282,7 +282,7 @@ def cli():
 @click.option(
     "--eval-type",
     help="Type of evaluation to run",
-    type=click.Choice(["mmlu", "mt-bench"]),
+    type=click.Choice(["mt-bench"]),
```

Comment:
We could get rid of this flag entirely IMO. If we don't please add `hidden=True` to not show it in the help.